latest/functions/Knative/logs-handler-function/README.md

```diff
@@ -1,6 +1,12 @@
 # Function Logs Handler
 
-> You can refer to the original article by visiting this: [以 Serverless 的方式实现 Kubernetes 日志告警](https://mp.weixin.qq.com/s/EZWYqtXJ7Cj-Yd7Fro6uyA)
+You can refer to the original article at: 
```

Comment:
"You can refer to the original article at:"
=>
For detailed description of this sample, you can refer to the following blogs: